### PR TITLE
Landing page info sections

### PR DESCRIPTION
**Pull request by rap1ds**
This PR implements new section to Landing Page `"kind": "info"`. It contains two main variations `"variation": "single_column"` and `"variation": "multi_column"`.

In `single_column` variation, the section can be with or without background and with or without Call To Action button.

In `multi_column` variation, the section can have 2 or 3 columns, with or without icons and with or without Call To Action button.

So all in all there are 12 combinations:

- **Info 1 - V1**: 1 column, background, button
- **Info 1 - V2**: 1 column, background, no button
- **Info 1 - V3**: 1 column, no background, button
- **Info 1 - V4**: 1 column, no background, no button
- **Info 2 - V1**: 3 columns, icons, buttons
- **Info 2 - V2**: 3 columns, no icons, buttons
- **Info 2 - V3**: 3 columns, icons, no buttons
- **Info 2 - V4**: 3 columns, no icons, no buttons
- **Info 3 - V1**: 2 columns, icons, buttons
- **Info 3 - V2**: 2 columns, no icons, buttons
- **Info 3 - V3**: 2 columns, icons, no buttons
- **Info 3 - V4**: 2 columns, no icons, no buttons

All the sections support mobile, tablet and desktop view sizes.

Screenshots:

screen shot 2016-06-22 at 15 04 43
screen shot 2016-06-22 at 15 04 54
screen shot 2016-06-22 at 15 05 04
screen shot 2016-06-22 at 15 05 14
screen shot 2016-06-22 at 15 05 23
screen shot 2016-06-22 at 15 05 29
screen shot 2016-06-22 at 15 05 36
screen shot 2016-06-22 at 15 05 44
screen shot 2016-06-22 at 15 05 53
screen shot 2016-06-22 at 15 05 59
screen shot 2016-06-22 at 15 06 06
screen shot 2016-06-22 at 15 06 14

### CODE REVIEW

File: app/assets/stylesheets/landing_page/mixins/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.landing-page__button {
+  $button-border-size: 2px;
+
+  font-family: ProximaNovaSoft-Bold;
+  font-size: 16px;
+  letter-spacing: 0px;
+  padding: (16px - $button-border-size) (32px - $button-border-size);
+  display: inline-block;
+
+  border-radius: 1000px; // Is there cleaner way to do this?
```
Comment (kpuputti) [inline app/assets/stylesheets/landing_page/mixins/_buttons.scss]: This is how I did it as well #toisaalla to get full roundness, you can just comment "high enough to get fully rounded button" etc.

File: app/assets/stylesheets/landing_page/sections/hero.scss [hunk 2]
```diff
@@ -14,6 +14,13 @@
   box-shadow: inset 0 0 0 999999px rgba(0, 0, 0, 0.5); /* is there any cleaner way? */
 }
 
+.hero__content {
+  max-width: 1366px;
```
Comment (kpuputti) [inline app/assets/stylesheets/landing_page/sections/hero.scss]: `1366px` looks like a magic number, is it some breakpoint and could it be in a variable?

File: app/assets/stylesheets/landing_page/sections/hero.scss [hunk 3]
```diff
@@ -14,6 +14,13 @@
...
+  margin: 0 auto;
+  padding: $dimension-mobile-margin;
+  margin-top: -4.44vh; /* -50px, when content height is 900px, 80vh */
```
Comment (kpuputti) [inline app/assets/stylesheets/landing_page/sections/hero.scss]: Be sure to test the cross browser behavior of the `vh` unit well: many known issues listed in http://caniuse.com/#feat=viewport-units

**Comment by kpuputti** [comment]
There are quite a lot of pixel values that might become hard to maintain or port, and maybe some of them might be a bit clearer in variables?

Otherwise, looks really clean! Be sure to test well since Autoprefixer is not in use and you have Flexbox and viewport units in use that have differences in browsers (especially mobile).

**Comment by rap1ds** [comment]
@kpuputti Good point! I tested with iOS7 and indeed the `vh` units didn't work. Also, I added prefixes for flexbox properties. Now iOS7 looks great!

**Pull request merged**